apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Logging/LoggingTest.cs

```diff
@@ -234,7 +257,32 @@ public class NoBufferWarningLoggerTestApplication : LoggerTestApplication
         public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         {
             SetupRoutes(app);
-            LoggerOptions loggerOptions = LoggerOptions.Create(LogLevel.Warning, BufferOptions.NoBuffer());
+            LoggerOptions loggerOptions = LoggerOptions.Create(LogLevel.Warning, null, BufferOptions.NoBuffer());
+            loggerFactory.AddGoogle(ProjectId, loggerOptions);
+        }
+    }
+
+    /// <summary>
+    /// An application that has a <see cref="GoogleLogger"/> with no buffer that will accept all logs
+    /// of level warning or above and has a <see cref="MonitoredResource"/> of 'build'.
+    /// </summary>
+    public class NoBufferResourceLoggerTestApplication : LoggerTestApplication
+    {
+        public static readonly MonitoredResource s_resource = new MonitoredResource
```

Comment:
Not that it matters too much, since this is in a test project, but I don't know if public fields should use the s_ naming. I think this should just be `Resource`